aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -188,12 +198,32 @@ class DispatchTable final {
     return operatorName_;
   }
 
+  // This function is a temporary hack, see comment at manuallyBoxedKernel_ member
+  void setManuallyBoxedKernel_(KernelFunction::InternalBoxedKernelFunction* func) {
+    TORCH_INTERNAL_ASSERT(!manuallyBoxedKernel_.has_value(), "Cannot set multiple manually boxed kernels for the same operator ", operatorName_);
+    manuallyBoxedKernel_ = func;
+
+    // make sure that all previously registered kernels get this manually boxed kernel
```

Comment:
Should we also check previously registered catchAllKernel_?
https://github.com/ljk53/pytorch/commit/2d7ede5f71693dca308b00570dcaee6c4148864e - this fixed a crash I ran into with mobile build.

Reply:
Sounds like this needs a rebase - catch-all kernels went away in #33817 (reducing the possibility this sort of bug being one of the motivations for the simplification)

Reply:
They didn't go away entirely, DispatchTable.h still has them.

@ljk53 : good catch, I added it to https://github.com/pytorch/pytorch/pull/34177